portfolio/src/main/webapp/index.html

```diff
@@ -21,6 +18,11 @@
     <div class="row">
 
      <div id="content">
+
+      <div id = "dynamicDescription">
```

Comment:
See below comment on hyphenated elements.

Plus, please keep consistent with the spacing
\<div id="dynamic-description">

Reply:
I'll fix the spacing and push a revision to the same branch